### PR TITLE
fixing a typo

### PR DESCRIPTION
**Pull request by HiltonGiesenow**

### CODE REVIEW

File: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ If your organization has gone through a rebranding, merger, or acquisition and n
 | Site customizations and embedded code | Absolute URLs embedded in SharePoint customizations aren't updated. | Edit customizations that contain absolute URLs and if necessary, change the URLs to the new domain name. |
 | Teams on the web and Teams desktop app | The first time someone tries to access the Files tab for a team or private channel, they'll receive an error. The tab will work for all users after that. | None |
 | Teams on the web and Teams desktop app | It can take 72 hours for meeting notes to work (for both current and previous meetings). | None |
-| Teams on the web and Teams desktop app | On the Files tab, any folders added with the "Add cloud storage" (which point to another SharePoint site) won't work.| Remove and readd the folders. |
-| Teams on the web and Teams desktop app | Document libraries added as a tab won't work. | Remove and readd the tab. |
+| Teams on the web and Teams desktop app | On the Files tab, any folders added with the "Add cloud storage" (which point to another SharePoint site) won't work.| Remove and read the folders. |
+| Teams on the web and Teams desktop app | Document libraries added as a tab won't work. | Remove and read the tab. |
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md:103]:
```suggestion
| Teams on the web and Teams desktop app | On the Files tab, any folders added with the "Add cloud storage" (which point to another SharePoint site) won't work.| Remove the folders and add them again. |
| Teams on the web and Teams desktop app | Document libraries added as a tab won't work. | Remove the tab and add it again. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @HiltonGiesenow, I think the word "readd" (re-add) was really confusing, so I have suggested changing the clause to make it clear. Please take a look and commit my suggested changes. Many thanks.

File: SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md (hunk 1 above)
Comment (HiltonGiesenow) [reply SharePoint/SharePointOnline/change-your-sharepoint-domain-name.md]: oh, I totally misread the text - I skimmed it and thought it was a typo for "read", but obviously it was "re *add*"! I think both spellings are actually correct, but with a dash is a bit easier to read I think? Updated the PR

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**